### PR TITLE
append the known issues with hub rejoining issue

### PR DESCRIPTION
**Pull request by yanmxa**

### CODE REVIEW

File: doc/README.md [hunk 1]
```diff
@@ -22,8 +22,7 @@ This document focuses on the features of the multicluster global hub.
     - [Import a regional hub cluster in default mode (tech preview)](#import-a-regional-hub-cluster-in-default-mode-tech-preview)
     - [Access the grafana](#access-the-grafana)
     - [Grafana dashboards](#grafana-dashboards)
-  - [Troubleshooting](./troubleshooting.md)
-  - [Development preview features](./dev-preview.md)
```
Comment (clyang82) [inline doc/README.md:26]: why do you remove the dev-preview section?

File: doc/README.md [hunk 2]
```diff
@@ -153,3 +152,7 @@ For common Troubleshooting issues, proceed [here](troubleshooting.md)
...
 
 2. We provide ability to drill down the `Offending Policies` dashboard when you click a datapoint from the `Policy Group Compliancy Overview` dashboard. But the drill down feature is not working for the first datapoint. You can click the second datapoint or after to see the drill down feature is working. The issue is applied to the `Cluster Group Compliancy Overview` dashboard as well.
+
+3. If you detach the regional hub and then rejoin the regional hub, The data (policies/managed clusters) might't be updated in time for the rejoined regional hub. You can fix this problem by doing any of the following:
+   1. Restart the `multicluster-global-hub-manager` pod on global hub.
+   2. Edit the ConfigMap `open-cluster-management-global-hub-system/incarnation-config` to modify `incarnation` with a A larger value. If it is the first time update, `1000` might be a good choice. If it is not the first update, make sure it is larger than the last change. And then restart the `multicluster-global-hub-agent` pod on the regional hub.
```
Comment (clyang82) [inline doc/README.md:158]: I suggest to make it easy for customer. Let us remove the second one. just restart the manager. That can make life simple.

File: doc/README.md (hunk 1 above)
Comment (yanmxa) [reply doc/README.md:26]: It is caused by the markdown plugin of my vscode. I'm trying to disable it.

File: doc/README.md (hunk 2 above)
Comment (yanmxa) [reply doc/README.md]: @clyang82 Done!

**Pull request merged**